mod_reforged/hooks/config/strings.nut

```diff
@@ -1290,6 +1291,20 @@ foreach (vanillaDesc in vanillaDescriptions)
 			]
 		}]
  	}),
+	RF_Inspire = ::UPD.getDescription({
+		Effects = [{
```

Comment:
Improve formatting by pushing the `{` to the next line because we have multiple entries in the array.